1-exercises/A-this/exercise-1.js

```diff
@@ -15,7 +15,9 @@ let person = {
     name: "Adrian",
     age: 32,
     currentAddress: "London",
-    celebrateBirthday: /* todo */
+    celebrateBirthday: function(){
```

Comment:
@Farnooshmo 
Hi Farnoosh, it is good solution.

The other way is you can try instead of
celebrateBirthday: function(){}

this one

celebrateBirthday() {}

because in this task celebrateBirthday is a method (method means a javascript function)